Checkpoint1/Program5/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Program5
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int max = 0;
```

Comment:
this assumes that the user will enter at least 1 positive number.
What if they enter all negative numbers?